smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -846,14 +846,63 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
       expectReportPlainResult(/*requestId=*/ 111, 'SUCCESS');
 
       // TAG_IFD_DEVICE_REMOVED is used with a value of 0 here. This is a no-op.
-      // Nothing will happen to the reader. 
+      // Nothing will happen to the reader.
       // The goal is just to test `SetAttrib` returning the `SUCCESS` value.
       await mockChromeApi
           .dispatchEvent(
               'onSetAttribRequested', /*requestId=*/ 111, readerHandle,
               TAG_IFD_DEVICE_REMOVED, /*data=*/ new Uint8Array(['0x0']))
           .$waitAndVerify();
     },
-  }
+  },
+
+  // Test Transmit success.
+  'testTransmit_success': async function() {
+    // Set up a reader and a card with a PIV profile.
+    await launchPcscServer(/*initialDevices=*/[{
+      'id': 123,
+      'type': SimulationConstants.GEMALTO_DEVICE_TYPE,
+      'cardType': SimulationConstants.COSMO_CARD_TYPE,
+      'cardProfile': SimulationConstants.CHARISMATHICS_PIV_TYPE
+    }]);
+
+    const sCardContext = await establishContext(/*requestId=*/ 66);
+    readerHandle = await connect(
+        /*requestId=*/ 67, sCardContext,
+        SimulationConstants.GEMALTO_PC_TWIN_READER_PCSC_NAME0,
+        chrome.smartCardProviderPrivate.ShareMode.SHARED,
+        /*preferredProtocols=*/ {'t1': true},
+        /*expectedProtocol=*/ chrome.smartCardProviderPrivate.Protocol.T1);
+
+    // Reply should contain the application identifier, followed by the status
+    // bytes that denote success: SW1=0x90, SW2=0x00. This corresponds to the
+    // fake reply built by `testing_smart_card_simulation.cc`.
+    expectReportDataResult(
+        /*requestId=*/ 111,
+        new Uint8Array([
+          0x61, 0x5C, 0x4F, 0x0B, 0xA0, 0x00, 0x00, 0x03, 0x08, 0x00, 0x00,
+          0x10, 0x00, 0x01, 0x00, 0x79, 0x07, 0x4F, 0x05, 0xA0, 0x00, 0x00,
+          0x03, 0x08, 0x50, 0x27, 0x50, 0x65, 0x72, 0x73, 0x6F, 0x6E, 0x61,
+          0x6C, 0x5F, 0x49, 0x64, 0x65, 0x6E, 0x74, 0x69, 0x74, 0x79, 0x5F,
+          0x61, 0x6E, 0x64, 0x5F, 0x56, 0x65, 0x72, 0x69, 0x66, 0x69, 0x63,
+          0x61, 0x74, 0x69, 0x6F, 0x6E, 0x5F, 0x43, 0x61, 0x72, 0x64, 0x5F,
+          0x50, 0x1A, 0x68, 0x74, 0x74, 0x70, 0x3A, 0x2F, 0x2F, 0x63, 0x73,
+          0x72, 0x63, 0x2E, 0x6E, 0x69, 0x73, 0x74, 0x2E, 0x67, 0x6F, 0x76,
+          0x2F, 0x6E, 0x70, 0x69, 0x76, 0x70, 0x90, 0x00
+        ]).buffer,
+        'SUCCESS');
+
+    // Send the SELECT command (the format is per NIST 800-73-4).
+    const selectCommand = new Uint8Array([
```

Comment:
nit: This is a "real" constant, so `SELECT_COMMAND` would be appropriate spelling for it.

Reply:
done